data.sql

```diff
@@ -13,19 +28,44 @@ VALUES
     ('Angemon', '2005-06-12', -45, 'true', 1, 2, 5),
     ('Boarmon', '2005-06-7', 20.4, 'true', 7, 2, 5),
     ('Blossom', '1998-10-13', 17, 'true', 3, 1, 4),
-    ('Ditto', '2022-05-14', 22, 'true', 4, 1, NULL)
+    ('Ditto', '2022-05-14', 22, 'true', 4, 1, NULL);
 
-INSERT INTO owners
-    (full_name, age)
+INSERT INTO vets
+    (vet_name, age, date_of_graduation)
 VALUES
-    ('Sam Smith', 34),
-    ('Jennifer Orwell', 19),
-    ('Bob', 45),
-    ('Melody Pond', 77),
-    ('Dean Winchester', 14),
-    ('Jodie Whittaker', 38)
+    ('William Tatcher', 45, '2000-04-23'),
+    ('Maisy Smith', 26, '2019-01-17'),
+    ('Stephanie Mendez', 64, '1981-05-04'),
+    ('Jack Harkness', 38, '2008-06-08');
 
-INSERT INTO species (name)
+INSERT INTO specialization
+    (vet_id, specie_id)
 VALUES
-	('Pokemon'),
-	('Digimon')
\ No newline at end of file
+    (1, 1),
+    (3, 1),
+    (3, 2),
+    (4, 2);
+
+INSERT INTO visits
+    (animal_id, vet_id, date_of_visit)
+VALUES
+    (1, 1, '2020-05-24'),
+    (1, 3, '2020-07-22'),
+    (2, 4, '2021-02-02'),
+    (3, 2, '2020-01-05'),
+    (3, 2, '2020-03-08'),
+    (3, 2, '2020-05-14'),
+    (3, 3, '2021-05-04'),
+    (5, 4, '2021-02-24'),
+    (6, 2, '2019-12-21'),
+    (6, 1, '2020-08-10'),
+    (6, 2, '2021-04-07'),
+    (7, 3, '2019-09-29'),
+    (8, 4, '2020-10-03'),
+    (8, 4, '2020-11-04'),
+    (9, 2, '2019-01-24'),
+    (9, 2, '2019-05-15'),
+    (9, 2, '2020-02-27'),
+    (9, 2, '2020-08-03'),
+    (10, 3, '2020-05-24'),
+    (10, 1, '2021-01-11');
```

Comment:
- [ ] (Optional) Imagine being the manager of a big database, it would be cumbersome hardcoding all values. you can automate it by doing this:

```
INSERT INTO visits (animal_id, vet_id, vist_date)
VALUES 
((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24'),
((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-07-22')
```